guides/release/testing/testing-application.md

```diff
@@ -44,9 +44,9 @@ module('Acceptance | posts', function(hooks) {
 
   test('should add new post', async function(assert) {
     await visit('/posts/new');
-    await fillIn('input.title', 'My new post');
-    await click('button.submit');
-    assert.equal(document.querySelector('ul.posts li').textContent, 'My new post');
+    await fillIn('[data-test-post-input-title]', 'My new post');
+    await click('[data-test-post-save]');
```

Comment:
**suggestion (non-blocking)**

I like to name test selectors according to what I (the user) can see on their screen. If the user can't see something (e.g. from an accessibility perspective), then I shouldn't be able to select that element for testing.

What do you think about the following names? You can imagine there's a `<label>` element with `Post Title` inside, and a `<button>` element with the label `Save`.

```suggestion
    await fillIn('[data-test-field="Post Title"]', 'My new post');
    await click('[data-test-button="Save"]');
```